public/video-ui/src/components/Video/Display.js

```diff
@@ -145,7 +151,8 @@ class VideoDisplay extends React.Component {
     return <div className="video__detailbox">
       <div className="video__detailbox__header__container">
         <header className="video__detailbox__header">Video Preview</header>
-        <Link className="button" to={`/videos/${this.props.video.id}/upload`}>
+        <Link className={"button " + (this.cannotMakeEdits() ? "disabled" : "")} to={`/videos/${this.props.video.id}/upload`}
```

Comment:
would it be more semantic to disable the element rather than preventing the default click handler?